website/versioned_docs/version-0.2.x/guides/interacting/using-apollo.md

```diff
@@ -10,6 +10,10 @@ It is **not necessary to use it** to [execute queries](queries.md) and [mutation
 
 The ComposeDB client can be used with the [Apollo client](https://www.apollographql.com/docs/react/api/core/ApolloClient) by creating a custom [Apollo link](https://www.apollographql.com/docs/react/api/link/introduction), as shown in the example below:
 
+Make sure you have the following packages installed, before running this code:
```

Comment:
Devs should read the Apollo docs for setup instructions, we shouldn't put specific instructions here